test/msw-api/context/async-response-transformer.mocks.ts

```diff
@@ -0,0 +1,20 @@
+import { ResponseTransformer, setupWorker, rest, context, compose } from 'msw'
+import base64Image from 'url-loader!../../fixtures/image.jpg'
+
+async function jpeg(base64: string): Promise<ResponseTransformer> {
```

Comment:
This is an example of a custom `jpeg` response transformer that appends a given Base64 image to the mocked response. It utilizes newly exported `compose` function, as well as the ability to make the transformer async.